Rajitha/google/google.css

```diff
@@ -0,0 +1,92 @@
+.topnav {
+  background-color: white;
+}
+.topnav a {
+  text-decoration: none;
+  color: black;
+  font-size: 15px;
+  font-weight: 250;
+  opacity: 0.90;
+  margin: 8px;
+  top: 10px;
+  text-align: left;
+  position: relative;   
+}
+.topnav a:hover{
+  text-decoration: underline;
+}
+.nav-right {
+    float: right;
+  }
+.center {
+  margin: 10px auto 15px;
+  margin-top: 150px;
+  display: block;
+  width: 17%;
+}
```

Comment:
look into margin-left / right  auto

Reply:
When I use margin-left, I am able to place logo in center but it is not responsive. It gets fixed to one place